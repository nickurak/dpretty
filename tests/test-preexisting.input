From 48c83c6a90172e30641994f0d232b48274961e3e Mon Sep 17 00:00:00 2001
From: Jeremy Nickurak <jeremy@nickurak.ca>
Date: Tue, 17 Mar 2020 13:13:08 -0600
Subject: [PATCH 1/2] file.py

---
 file.py | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 file.py

diff --git a/file.py b/file.py
new file mode 100644
index 0000000..4e8dd71
--- /dev/null
+++ b/file.py
@@ -0,0 +1,2 @@
+def func():
+       print("too much indent")
-- 
2.24.1


From d29212593491d13ca601c7970a5c871ec52422de Mon Sep 17 00:00:00 2001
From: Jeremy Nickurak <jeremy@nickurak.ca>
Date: Tue, 17 Mar 2020 13:15:43 -0600
Subject: [PATCH 2/2] another poor commit

---
 file.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/file.py b/file.py
index 4e8dd71..050dcb4 100644
--- a/file.py
+++ b/file.py
@@ -1,2 +1,5 @@
 def func():
-       print("too much indent")
+    print("too much indent")
+
+def func2():
+       print("another poorly indented func")
-- 
2.24.1

