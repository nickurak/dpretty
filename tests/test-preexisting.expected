commit 1cd833a1cee356b9c2e5d2a8d74d0a1c393b0dff
Author: Dpretty tests <dpretty-tests@nonexistant.invalid>
Date:   Wed Mar 18 09:56:35 2020 -0600

    init

commit a0a6d272b3c7778796f8b7bbb91db9156c3ce0eb
Author: Jeremy Nickurak <jeremy@nickurak.ca>
Date:   Tue Mar 17 13:13:08 2020 -0600

    file.py

diff --git a/file.py b/file.py
new file mode 100644
index 0000000..4e8dd71
--- /dev/null
+++ b/file.py
@@ -0,0 +1,2 @@
+def func():
+       print("too much indent")

commit bf63c847f8daa43cb5d6c8cee169db06c3117c0d
Author: Dpretty Formatter <dpretty-formatter@nonexistant.invalid>
Date:   Wed Mar 18 15:56:37 2020 +0000

    Auto-applying /usr/local/bin/dpretty.sh to these files first, to make subsequent commits easier to read:
    
    file.py

diff --git a/file.py b/file.py
index 4e8dd71..207af59 100644
--- a/file.py
+++ b/file.py
@@ -1,2 +1,2 @@
 def func():
-       print("too much indent")
+    print("too much indent")

commit 4925498c50d3d6093fc985af266c314a2f55cc58
Author: Jeremy Nickurak <jeremy@nickurak.ca>
Date:   Tue Mar 17 13:15:43 2020 -0600

    another poor commit

diff --git a/file.py b/file.py
index 207af59..8a95f33 100644
--- a/file.py
+++ b/file.py
@@ -1,2 +1,6 @@
 def func():
     print("too much indent")
+
+
+def func2():
+    print("another poorly indented func")
